### DISCUSSION TITLE
Not related to technicolor: possible to get access from quantenna shell to other soc(realtek)

### DISCUSSION THREAD

**Discussion by Vargaskri** [hack-technicolor/hack-technicolor#241] [Q&A]
Hello. 

I'm a little lost and i know the people here are good at this. So i hope you can help.

I have got a root shell on quantenna soc. But i want to reach realtek soc where all the goodies are. 
Cant interrupt U-boot on realtek uart with nand glitching and i don't know the login if i manage to glitch the kernel to panic. 
So right now im stuck with shell on quantenna uart only.

The CPUs are talking trough something called QTN-EMAC?
Have no SSH, cant mount USB

Can i reach the NAND somehow? Make new MTD partitions? I would have read the NAND from u-boot if the watchdog havent been restarting the system every 15 sec...
Anyone have any ideas. 
Im still novice on this stuff and sorry if im not making any sense. 

I have bootlogs on my github. [https://github.com/Vargaskri/Icotera-i4882](url)

**Comment by LuKePicci** [comment]
No way, unless that qtn-emac protocol is interpreting some received commands at the Realtek side as shell commands. This would require you to know pretty much of what's happening on that side to check for exploitable command injection holes.
